addons/survey/views/survey_question_views.xml

```diff
@@ -29,42 +28,50 @@
                             <field name="question_type" widget="radio" attrs="{'required': [('is_page', '=', False)]}" />
                         </group>
                         <group>
-                            <div class="col-lg-6 offset-lg-3 o_preview_questions">
+                            <div class="col-lg-6 offset-lg-3 o_preview_questions" style="width:100%;height:101px">
```

Comment:
A few things here:

- We try to avoid custom styling in form as much as possible.
- If it's not possible to avoid id, we try to use bootstrap classes as much as possible (e.g here: "w-100")
- In any case, you should (almost) never write style attributes, use scss instead
- 101px height is oddly specific... what are you trying to do here?

Reply:
I don't know why i used 101px. I changed it to 100px.